test/aries-js-worker/test/messaging/messaging.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import {newDIDExchangeClient} from "../didexchange/didexchange_e2e.js";
+
+const basicMsgType = "https://didcomm.org/basicmessage/1.0/message"
+const basicMsgSvcName = "basic-msg-svc-demo"
+const sampleMsg = {
+    "@id": "1d9a1589-9d7b-4308-9fab-8ee9730720c2",
+    "@type":basicMsgType,
+    "~l10n":{"locale":"en"},
+    "sent_time":"2020-03-05T16:59:47.489789-05:00",
+    "content":"Your hovercraft is full of eels."
+}
+
+// scenarios
+describe("Basic Messaging", function () {
+    this.timeout(5000);
+
+    let connections, didexClient, sender, receiver
```

Comment:
`connections` could be an object with distinct and descriptive properties instead of just a list. Eg.:

```js
{
    receiver: "",
    sender: ""
}
```

Reply:
here sender & receiver are aries instances
connections renamed to destinationConnID